osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -139,7 +140,7 @@ public abstract partial class DrawableHitObject : PoolableDrawableWithLifetime<H
 
         protected override bool RequiresChildrenUpdate => true;
 
-        public override bool IsPresent => base.IsPresent || (State.Value == ArmedState.Idle && Clock?.CurrentTime >= LifetimeStart);
+        public override bool IsPresent => base.IsPresent || (State.Value == ArmedState.Idle && Clock.IsNotNull() && Clock.CurrentTime >= LifetimeStart);
```

Comment:
I believe this is either a compile error or a R# error. Try decompiling `Drawable` by following the reference here. It will have `#nullable disable` and #nullable enable` annotations littered all throughout it.